tendrl/ceph_integration/objects/definition/ceph.yaml

```diff
@@ -164,8 +164,8 @@ namespace.ceph:
             mandatory:
               - Rbd.name
               - Rbd.size
-            optional:
               - Rbd.pool_id
```

Comment:
Rbd.pool_id is must because librbd requires pool_name compulsory, so if we pass pool_id then only pool_name is identified and passed.